src/OrchardCore.Modules/OrchardCore.Themes/Views/Items/ThemeEntry-Descriptions.SummaryAdmin.cshtml

```diff
@@ -0,0 +1,8 @@
+@using OrchardCore.DisplayManagement.Views
+@using OrchardCore.Themes.Models
+
+@model ShapeViewModel<ThemeEntry>
+
+<span>
+    @Model.Value.Extension.Manifest.Description
+</span>
```

Comment:
Do we need the `span`? It'll be inside a `p`.

Reply:
It does not have to be in a span. But I think with spans you can style it easily if you need to.

Reply:
A span in itself only helps little with that. Then at least at a class to it, please.

Reply:
@Piedone 
It helps "if you want to select it". We don't need to add special class. 

It is much easier to style the description only when there is a span. for example, if someone want to style the description only then can do this.

```
.card-body .card-text > span {
 ///
}
```

Otherwise, the following will target everything not just description.

```
.card-body .card-text {
 ///
}
```

Again, just slightly better.

Do you see an issue of adding it?

Reply:
Yeah, I think it's clutter in itself. You can even select that text without a span if you really want it, but it's easier with a class. So, if you're going to change the markup to make it easier to select that next, then I'd make it really easy (and in line with the other elements in these templates).

Reply:
okay. I added a class.